internal/pkg/service/stream/storage/level/local/writer/volume/writer_test.go

```diff
@@ -212,7 +226,6 @@ ghi,jkl,789
 {"level":"debug","message":"file synced"}
 {"level":"debug","message":"chain closed"}
 {"level":"debug","message":"closed file"}
-{"level":"info","message":"closed volume"}
```

Comment:
Volume is closed after the test by the `t.Cleanup`, so it is not part of the checked logs.